README.md

```diff
@@ -30,16 +30,147 @@ MIDDLEWARE = [
 ]
 ```
 
-Finally, create a layout which exposes `{% block inertia %}{% endblock %}` in the body and set the path to this layout as `INERTIA_LAYOUT` in your `settings.py` file.
+#### Vite Installation
 
-Now you're all set!
+Before you start it's recommended to start a new app. For this example we created a app called `core`.
+
+Install the following python packages via pip
+```bash
+pip install django-vite whitenoise
+```
+
+Create a layout template file
+```html
+{% load django_vite %}
+<!DOCTYPE  html>
+<html  lang="en">
+<head>
+	<meta  charset="UTF-8"  />
+	<meta  http-equiv="X-UA-Compatible"  content="IE=edge"  />
+	<meta  name="viewport"  content="width=device-width, initial-scale=1.0"  />
+	{% vite_hmr_client %}
+	{% vite_asset 'main.ts' %}
+	
+	<title>Inertia Layout</title>
+</head>
+
+<body>
+	{% block inertia %}{% endblock %}
+</body>
+</html>
+```
+
+Add the Inertia app to your `INSTALLED_APPS` in `settings.py`
+```
+INSTALLED_APPS = [
+  # django apps,
+  'inertia',
+  'django_vite',
+  # your project's apps,
+]
+```
+
+Add whitenoise middleware in `settings.py`
+```python
+MIDDLEWARE = [
+    # ...
+    "django.middleware.security.SecurityMiddleware",
+    "whitenoise.middleware.WhiteNoiseMiddleware",
```

Comment:
Perhaps better to keep the whitenoise config as an example rather than within the vite and inertia setup?